docs/modules/ROOT/pages/configuration/enable-observability.adoc

```diff
@@ -4,8 +4,13 @@ KubeArchive is instrumented to emit observability data, specifically metrics and
 traces. This can be enabled and configured with two environment
 variables present on all of the KubeArchive components:
 
-* `KUBEARCHIVE_OTEL_ENABLED`: one of "true" or "false". Defaults to "false". It
-   controls if observability data is sent to `OTEL_EXPORTER_OTLP_ENDPOINT`.
+* `KUBEARCHIVE_OTEL_MODE`: one of "enabled" or "delegated". Defaults to an empty string (`""`). It
+   controls when observability data is sent to `OTEL_EXPORTER_OTLP_ENDPOINT`:
+** Empty string (`""`): do not send traces or metrics.
```

Comment:
What about having a explicit `disabled` instead of an empty value?

Reply:
Great, I believe it's less error prone to see an explicit value in the configuration :)

Reply:
How can you have `disabled` on a boolean value?  Or is this only for `KUBEARCHIVE_OTEL_MODE`?

Reply:
This is only for `KUBEARCHIVE_OTEL_MODE`, which replaces the old (boolean) value.